tensilelite/Tensile/Components/SIA.py

```diff
@@ -618,15 +618,13 @@ def prepareLWInstToSched(writer, kernel, numLocalWritesPerSched):
     #################
     itemsLWToSched = list(writer.codes.localWriteA.items()) + list(writer.codes.localWriteB.items())
     if kernel["PrefetchGlobalRead"] == 2:
-        # PrefetchGlobalRead + DirectToLds case, need to add dummy list to insert global read
+        # PrefetchGlobalRead + DirectToLds/DirectToVgpr case, need to add dummy list to insert global read
         tmpList = []
-        numItemsBeforeStoreC = 0
```

Comment:
I removed numItemsBeforeStoreC.
This is only for StoreCInUnroll (only in Tensile) and will not be necessary.